src/hotspot/share/runtime/arguments.hpp

```diff
@@ -107,8 +108,9 @@ class SystemProperty : public PathString {
   bool internal() const               { return _internal; }
   SystemProperty* next() const        { return _next; }
   void set_next(SystemProperty* next) { _next = next; }
+  bool writeable() const              { return _writeable; }
 
-  bool is_readable() const {
+  bool readable() const {
```

Comment:
Might be better/simpler to keep is_readable and change to is_writable(), as that avoids changing other files.

Reply:
There's also `internal()`, which would need to be changed to `is_internal()` as well for consistency. I chose to change `is_readable()` to `readable()` so I just needed to change one name.

Reply:
Okay.